<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Related work on the analysis of scRNA-seq count data</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/sandstone.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-1.1/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-1.1/highlight.js"></script>
<link href="site_libs/font-awesome-4.5.0/css/font-awesome.min.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs && document.readyState && document.readyState === "complete") {
   window.setTimeout(function() {
      hljs.initHighlighting();
   }, 0);
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 61px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 66px;
  margin-top: -66px;
}

.section h2 {
  padding-top: 66px;
  margin-top: -66px;
}
.section h3 {
  padding-top: 66px;
  margin-top: -66px;
}
.section h4 {
  padding-top: 66px;
  margin-top: -66px;
}
.section h5 {
  padding-top: 66px;
  margin-top: -66px;
}
.section h6 {
  padding-top: 66px;
  margin-top: -66px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">scRNA-seq workflow</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/jhsiao999/singleCell-workflow">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Related work on the analysis of scRNA-seq count data</h1>

</div>


<p><strong>Last updated:</strong> 2017-02-15</p>
<p><strong>Code version:</strong> 466ddd3</p>
<hr />
<p>Please note that this list is not exhaustive and only includes the work relevant to our methods under development.</p>
<div id="list-of-workflows" class="section level4">
<h4>List of workflows</h4>
<ol style="list-style-type: decimal">
<li><p>Aaron T. L. Lun, Davis J. McCarthy, John C. Marioni. A step-by-step workflow for low-level analysis of single-cell RNA-seq data [version 1; referees: 5 approved with reservations]. F1000Research 2016, 5: 2122. <a href="doi:10.12688/f1000research.9501.1" class="uri">doi:10.12688/f1000research.9501.1</a></p></li>
<li><p>Davis J. McCarthy, Kieran R. Campbell, Aaron T. L. Lun, Quin F. Willis. scater: pre-processing, quality control, normalisation and visualization of single-cell RNA-seq data in R. bioRxiv 069633; <a href="doi:10.1101/069633" class="uri">doi:10.1101/069633</a></p></li>
<li><p>Minzhe Guo, Hui Wang, Steven Potter, Jeffrey A. Whitsett, Yang Xu. SINCERA: A pipeline for single-cell RNA-seq profiling analysis. PLoS Computational Biology 2015;11(11):e1004575. <a href="doi:10.1371/journal.pcbi.1004575" class="uri">doi:10.1371/journal.pcbi.1004575</a></p></li>
</ol>
</div>
<div id="filtering" class="section level4">
<h4>Filtering</h4>
<ol style="list-style-type: decimal">
<li>Tomislav Ilicic, Johng Kyoung Kim, Aleksandra A. Kolodziejczyk, Frederik Otzen Bagger, Davis James McCarthy, John C. Marioni, Sarah A. Teichmann. Classification of low quality cells from single-cell RNA-seq data. Genome Biology 2016; 17:29. <a href="doi:10.1186/s13059-016-0888-1" class="uri">doi:10.1186/s13059-016-0888-1</a></li>
</ol>
</div>
<div id="normalization-and-batch-effect-correction" class="section level4">
<h4>Normalization and batch effect correction</h4>
<ol style="list-style-type: decimal">
<li><p>Aaron T. L. Lung, Bach Karsten, John C. Marioni. Pooling across cells to normalize single-cell RNA sequencing data with many zero counts. Genome Biology 2016; 17(1):75. <a href="doi:10.1186/s13059-016-0947-7" class="uri">doi:10.1186/s13059-016-0947-7</a></p></li>
<li><p>Bo Ding, Lina Zheng, Yun Zhu, Nan Li, Haiyang Jia, Rizi Ai, Andre Wildberg, Wei Wang. Normalization and noise reduction for single cell RNA-seq experiments. Bioinformatics 2015;31(13):2225-27. <a href="doi:10.1093/bioinformatics/btv122" class="uri">doi:10.1093/bioinformatics/btv122</a></p></li>
</ol>
</div>
<div id="cell-cycle-phase" class="section level4">
<h4>Cell cycle phase</h4>
<ol style="list-style-type: decimal">
<li><p>Scialdone A, Natarajan KN, Saraiva LR, Proserpio V, Teichmann SA, Stegle O, Marioni JC, Buettner F. Computational assignment of cell-cycle stage from single-cell transcriptome data. Methods 2015; 85:54-61. <a href="doi:10.1016/j.ymeth.2015.06.021" class="uri">doi:10.1016/j.ymeth.2015.06.021</a></p></li>
<li><p>Martin Barron, Jun Li. Identifying and removing the cell-cycle effect from single-cell RNA-sequencing data. Scientific reports 2016;6:33892. <a href="doi:10.1038/srep33892" class="uri">doi:10.1038/srep33892</a></p></li>
</ol>
</div>
<div id="differential-analysis" class="section level4">
<h4>Differential analysis</h4>
<ol style="list-style-type: decimal">
<li><p>Trung Nghia Vu, Quin F. Willis, Krishna R. Kalari, Nifang Niu, Liewei Wang, Mattias Rantalainen, Yudi Pawitan. Beta-poisson model for single-cell RNA-seq data analyses. Bioinformatics 2016;32(14):2128-35. <a href="doi:10.1093/bioinformatics/btw202" class="uri">doi:10.1093/bioinformatics/btw202</a></p></li>
<li><p>Vallejos CA, Richarson S, Marioni JC. Beyond comparisons of means: understanding changes in gene expression at the single-cell level. Genome Biology 2016; 17(1):70. <a href="doi:10.1186/s13059-016-0930-3" class="uri">doi:10.1186/s13059-016-0930-3</a></p></li>
<li><p>Finak G, McDavid A, Yajima M, Deng J, Gersuk V, Shalek AK, Slichter CK, Miller HW, McElrath MJ, Prlic M, Linsley PS. MAST: a flexible statistical framework for assessing transcriptional changes aand characterizing heterogeneity in single cell RNA sequencing data. Genome Biology 2015;1691:278. <a href="doi:10.1186/s13059-015-0844-5" class="uri">doi:10.1186/s13059-015-0844-5</a></p></li>
</ol>
</div>

<hr>
<p>
    This <a href="http://rmarkdown.rstudio.com">R Markdown</a> site was created with <a href="https://github.com/jdblischak/workflowr">workflowr</a>
</p>
<hr>

<!-- To enable disqus, uncomment the section below and provide your disqus_shortname -->

<!-- disqus
  <div id="disqus_thread"></div>
    <script type="text/javascript">
        /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
        var disqus_shortname = 'rmarkdown'; // required: replace example with your forum shortname

        /* * * DON'T EDIT BELOW THIS LINE * * */
        (function() {
            var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
            dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
            (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
        })();
    </script>
    <noscript>Please enable JavaScript to view the <a href="http://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
    <a href="http://disqus.com" class="dsq-brlink">comments powered by <span class="logo-disqus">Disqus</span></a>
-->



</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
