---
title: "An example workflow: our recent technical pilot study"
output:
  html_document:
    toc: false
---

**Last updated:** `r Sys.Date()`

**Code version:** `r workflowr::extract_commit(".", 1)$sha1`

---


Our previous workflow for generating count tables from raw sequence reads is summarized as below. 


## Steps

1. Generate index genomes for mapping sequence reads.

* For both endogeneous and spike-in controls

2. Quality control and trimming:

* QC sequence reads
* Extract UMIs
* Trimming 3' end of reads

3. Map reads to the index genomes.

* Mapping reads
* Sort, index and combine bam files

4. Generate gene-level read count and molecule-count.

* Count reads per gene
* Count molecules per gene


## Tools

(Need to confirm the versions)

* umitools
* sickel version 1.33
* Subread version 1.5.0-p1
* samtools version 0.1.18-dev
* UMI-tools (versione0ade5d)


