---
title: Tools that we may use
output:
  html_document:
    toc: false
---

**Last updated:** `r Sys.Date()`

**Code version:** `r workflowr::extract_commit(".", 1)$sha1`

---


0. General purposes

    * [bioawk](https://github.com/lh3/bioawk) is an extention of `awk` with additional support for biological data.
    * [snakemake](https://snakemake.readthedocs.io/en/latest/)

1. Generate index genomes for mapping sequence reads.

    * [Subread](http://subread.sourceforge.net/) 
        * Endogeneous
        * Worm 
        * Fly


2. Quality control and trimming:

    * [fastqc](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) for QC sequencing reads.
    * [umitools](https://github.com/brwnj/umitools/releases/tag/v2.1.1) was used in our pilot project to extract UMIs.
    * [UMI-tools](https://github.com/CGATOxford/UMI-tools) will be used in the current project for extracting UMIs.
    * [sickle](https://github.com/najoshi/sickle) is used for trimming 3' end of reads.


3. Map reads to the index genomes.

    * [Subjunc](http://subread.sourceforge.net/) is used to map sequencing reads. It is a part of the [Subread](http://subread.sourceforge.net/) suite of softwares. 
    * [samtools](http://samtools.sourceforge.net/) is used to process bam files.

4. Generate gene-level read count and molecule-count.

    * [featureCounts](http://subread.sourceforge.net/) is used to count the number of reads or molecules per gene. It is a part of the [Subread](http://subread.sourceforge.net/) suite of softwares. 




